waku/wdd.md

```diff
@@ -0,0 +1,75 @@
+# Waku DNS Discovery
+
+> Version: 0.1.0 (Draft)
+>
+> Authors: Dean Eigenmann <dean@status.im>
+
+## Table of Contents
+
+1. [Abstract](#abstract)
+2. [Specification](#specification)
+    1. [Records](#records)
+    2. [Traversing](#traversing)
+3. [Footnotes](#footnotes)
+4. [Copyright](#copyright)
+
+## Abstract
+
+In this specification, we describe the DNS discovery protocol used by [Waku](./waku.md). 
+
+## Specification
+
+This specification is based on the DNS discovery proposal for Ethereum<sup>1</sup>. It traverses a merkle tree that is stored in the DNS records of a chosen domain.
+
+### Records
+
+The node lists are encoded as merkle trees and stored within DNS TXT records. The root of the tree is a `TXT` record:
+
+```
+tree-root:v1 r=<root> l=<link-root> seq=<sequence-number> sig=<signature>
+```
+
+- `root`: the root hash of the subtree.
```

Comment:
Can we add a basic graph of how this tree is visualized? I'm also not 100% sure I understand rationale of making this a tree instead of a list

Reply:
EIP answers this in

 >Why does the link subtree exist?

Should be briefly mentioned in rationale imo, so it is self contained